webviz_subsurface/plugins/_simulation_time_series/utils/from_timeseries_cumulatives.py

```diff
@@ -141,3 +146,25 @@ def rename_vector_from_cumulative(vector: str, as_rate: bool) -> str:
         if as_rate
         else f"INTVL_{vector}"
     )
+
+
+# pylint: disable=too-many-return-statements
+def datetime_to_intervalstr(date: datetime.datetime, freq: Frequency) -> Optional[str]:
+    if date is None:
+        return date
```

Comment:
```suggestion
    if date is None:
        return None
```
Maybe slightly more readable?